api/enterprise-api-ref/status-settings.markdown

```diff
@@ -138,7 +158,21 @@ administrator.
     Threshold in minutes that hosts are unreachable before they are considered a health issue.
 * **sameHostsNumberOfRuns** *(integer)*
     Number of samples used to identify a duplicate identity. Default value is 3.
-
+* **enforce2FA** *(boolean)*
+    Determines if two-factor authentication (2FA) is mandatory for all users.
+    If set to `true`, users must enable 2FA; otherwise, they will be locked within 48 hours after the first login.
```

Comment:
Why the duplication of these fields? Why not just refer to the above block with something like

```
See "Get settings" section for field descriptions
```

And maybe just include a single line for each field that is available/required.
It is a bit curious that the default values are mentioned above in the "Get settings" section. It would seem most appropriate here when using "Update settings".

Reply:
Done, pointed from the get settings output to the fields of update section